cmd/samples/http/sender-with-custom-client/main.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+	"github.com/google/uuid"
+	"github.com/kelseyhightower/envconfig"
+)
+
+const (
+	count = 1000
+)
+
+type envConfig struct {
+	// Target URL where to send cloudevents
+	Target string `envconfig:"TARGET" default:"https://localhost:8080" required:"true"`
+}
+
+func main() {
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		log.Printf("[ERROR] Failed to process env var: %s", err)
+		os.Exit(1)
+	}
+	os.Exit(_main(os.Args[1:], env))
+}
+
+// Example is a basic data struct.
+type Example struct {
+	Sequence int    `json:"id"`
+	Message  string `json:"message"`
+}
+
+func _main(args []string, env envConfig) int {
+	source, err := url.Parse("https://github.com/cloudevents/sdk-go/cmd/samples/sender")
+	if err != nil {
+		log.Printf("failed to parse source url, %v", err)
+		return 1
+	}
+
+	// Read certificate and key information from the environment for HTTPS
+	clientCert := os.Getenv("CLIENT_CERT")
+	clientKey := os.Getenv("CLIENT_KEY")
```

Comment:
Can you migrate this to use the envconfig, noted above.

Reply:
done